<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe Script</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>System of Equations</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::AppointmentNew"/>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-background-color: rgb(232, 232, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLable
{
	color: rgb(38, 69, 117);
	font: 700 18pt &quot;Times New Roman&quot;;
}</string>
      </property>
      <property name="text">
       <string>System of Equetions: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 199, 146);
font: 9pt &quot;Segoe Script&quot;;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <property name="text">
       <string>   Row</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe Script&quot;;
background-color: rgb(0, 109, 80);
background-color: rgb(0, 112, 192);
background-color: rgb(0, 199, 146);</string>
      </property>
      <property name="text">
       <string>   Col</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe Script&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="solveButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe Script&quot;;
background-color: rgb(0, 199, 146);</string>
      </property>
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="clearmatrix">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe Script&quot;;
background-color: rgb(0, 199, 146);</string>
      </property>
      <property name="text">
       <string>Clear ðŸ§¹</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QTextEdit" name="outputTextEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
